.github/workflows/phpunit-tests.yml

```diff
@@ -256,22 +269,37 @@ jobs:
       - name: Install WordPress
         run: npm run env:install
 
+      - name: Run slow PHPUnit tests on PHP 5.6
+        if: ${{ matrix.split_slow }}
+        run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c ${{ env.PHPUNIT_CONFIG }} --group ${{ env.SLOW_TESTS }}
+
+      - name: Run PHPUnit tests for single site excluding slow tests on PHP 5.6
+        if: ${{ matrix.php == '5.6.20' && ! matrix.split_slow && ! matrix.multisite }}
+        run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c ${{ env.PHPUNIT_CONFIG }} --exclude-group ${{ env.SLOW_TESTS }},ajax,ms-files,ms-required
+
+      - name: Run PHPUnit tests for Multisite excluding slow tests on PHP 5.6
+        if: ${{ matrix.php == '5.6.20' && ! matrix.split_slow && matrix.multisite }}
+        run: npm run test:${{ env.PHPUNIT_SCRIPT }} -- --verbose -c ${{ env.PHPUNIT_CONFIG }} --exclude-group ${{ env.SLOW_TESTS }},ajax,ms-files,ms-excluded,oembed-headers
+
       - name: Run PHPUnit tests
+        if: ${{ matrix.php != '5.6.20' }}
```

Comment:
Would it be better to use `${{ matrix.php < '7.0' }}` check instead? Only for the same reason of backporting mentioned above.